### ISSUE TITLE
Airdrop farmer safes associated with address 0xac

### ISSUE THREAD

**Issue by p3rcy-eth** [safe-global/safe-user-allocation-reports#130]
# Related Safe Addresses

(At least) 46 safes were created by an airdrop farmer.

0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956
0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce
0xc3011a19b221d2571b94e85b3079c9092b437b73
0x11d61ca55bed7e67f509a054e68958b911e4e163
0x2a5d746c8b5b099bc32f17c0862b52dde9e4a7fa
0xc3011a19b221d2571b94e85b3079c9092b437b73
0x11d61ca55bed7e67f509a054e68958b911e4e163
0x2a5d746c8b5b099bc32f17c0862b52dde9e4a7fa
0xf8585ee97a6edede2d6438f60b05e543dc5c703b
0xc2284bfcbceea21f1e914e67f796583d8b1cec1b
0xed922a25aa6f8ffa8fb99343e5b0c9a90c517193
0x2696b20a5e3bd09703f825d449fb973ed18315b9
0x74071154a47cb452b53d1c8940ebfe86620b515a
0x0ed9eef0d135ea684067b2cf7c9a8809200ffc4d
0xe40d6d04a6b02de42537ed437e3d4acfaed60451
0xd4b613b77c713cc9f500c05da1f4be33342e7354
0x782de266cc0524581bd6b6d0bfe3ddcab3ac1998
0x9116d17e4e2996ac620985d3c81c3df9aae9757a
0x625cc1d6280d8288ddeee4e313d309e96d3dcbdf
0x8ecab80540b4e764e65a0586d99c9ebb351ecdfe
0x54262b056000332bd82793f7154c71966010554b
0xc6d6e408dc0a01cb26ddd2f8316e9f4b3e9a1e75
0x678141b710e680380167aeb0fcbd6010213fe339
0x44d026e6952e42b2f3d6e881fb9619f6e111b63f
0x210c0642ac42962ed4609b8433f6a263d0f3d1e8
0xd3f49e496d2ba3041739fcb8a644c29ca6a1af42
0x5b332dca77ea2eb9ad281b313de071b6d14330de
0x7d4d1c22edce3d69c6212e6dad9c51e6af3137fd
0xf2444be94cc78be648f5d90b686c0e5a282cacec
0x77a8f147a78873bcd49acbc96b533440ca6c2c82
0xc33959fe31ce88fd8b4fc60579dd6ab6998fd575
0x9f08511f0d88a360801a837f47700d8207b7456d
0xcf066abe1e200919882a55aa026b7549bd78e632
0x85d150e0779b8bad89fd8885bece16f34026a7b2
0xd6e188fc0273ea597c7ba350a5d411b2c1816987
0x296428d8868d51e81fd28fcf208757713975f369
0x26cfd0af2f9d7a9cd0e05ea7d92382d9189f5897
0xcf36a04ede4f3ea8f11c524a3073d9c8dca91c5d
0x228ea0ee7c976a2b83bb7d95c3d801beefd4f866
0x29d88b0a3dfb8971e13853b6dcfcac1124e54551
0x119d4499e697a371b3804440bb6cd406dd1388db
0xb803e9bb25b5d1e0c536c18f8069e55f3f910fca
0x7cebc0abe652a2d84f5cd0aecff6604c1088a10c
0x1f21f4c79aa7b46933ca6f6811e321f4170cb296
0x7cf59d3a059d4e410091c2303d0d01b304b3c22a
0x731598be46067f3e5724571e75ec96a7e41ab8df

# Reasoning

All these safes have one thing in common: they were created by an address that newly received tiny funds (usually about 0.01-0.15 ETH) from OKEx 3 from 8/12/2022 to 8/18/2022 (UTC). After safe creation, those new addresses will make some deposits of ETHs, make some "Exec Transaction" interactions with the safe, and then transfer the remaining funds to **0xaC40B1B9afFdf358Ff4F268085562B1EFAb8a473** (https://etherscan.io/address/0xac40b1b9affdf358ff4f268085562b1efab8a473). On the Etherscan page, we can clearly observe that tiny amount of ETH are converging from many different addresses, and if we further check those addresses 1 by 1, we can find that all those addresses created a safe and made some 0 value "Exec Transaction" at about the same time.

# Methodology

The reported safe addresses were discovered by checking the airdrop qualified safes from the csv file. I came across one of the safes, saw that there are few txns (creation and 0 value txns), and decided to manually trace the safe creator's addresses to see if someone is using this method to create multiple safes.

# Safe Address

eth:0xd5c6686A259D3C2625927Ba9E4db37D000433b92

**Comment by tschubotz** [comment]
I checked the first 2 Safes.
- https://etherscan.io/address/0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956#internaltx.
- https://etherscan.io/address/0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce

> and then transfer the remaining funds to **0xaC40B1B9afFdf358Ff4F268085562B1EFAb8a473**

At least for the 2 addresses above, that's not true. Please elaborate.

**Comment by p3rcy-eth** [comment]
> 

(1) The creator of safe **0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956 (safe1)** is **0xE3e959BE4af88296ccf67134939f2609a58Eb9f0 (address1)**. The txn that transfers funds from **address1** to the **main address** is https://etherscan.io/tx/0xc3b4b922c7d0e90f32a833f84e59f89367ccdcc2cdd8c4c9fc66e895594ca3b7, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

(2) The creator of safe **0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce (safe2)** is **0x89E7781D7bdbbf92Deb5d75574188AA2fe6B54AC (address2)**. The txn that transfers funds from **address2** to the **main address** is https://etherscan.io/tx/0xb9aecf33cb1b058fec076fbed87bde23d6723e540a39b0e9c2e1fe4a64d72fbc, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

(3) The creator of safe **0xc3011a19b221d2571b94e85b3079c9092b437b73 (safe3)** is **0xE4c19E77073d9a22840DB594E6dFcF282748EDFf (address3)**. The txn that transfers funds from **address3** to the **main address** is https://etherscan.io/tx/0xd2fe66829f734f5e73d651664e28cd2cbdeeb3faaeec0a1057078e2d451cac97, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

To sum up, certain addresses used for airdrop farming might not exhibit completely same movement patterns. For example, for **safe2** and **address2**, after the txn that transfers funds from **address2** to the **main address**, the airdrop farmer used another address to transfer some ETH into **address2** and made some interactions with **safe2**. But in all, airdrop farming is obvious. I can list all safe addresses, creator addresses, and farming pattern etherscan links if you need. Let me know if you need further details.

**Comment by p3rcy-eth** [comment]
> I checked the first 2 Safes.
> 
>     * https://etherscan.io/address/0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956#internaltx.
> 
>     * https://etherscan.io/address/0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce
> 
> 
> > and then transfer the remaining funds to **0xaC40B1B9afFdf358Ff4F268085562B1EFAb8a473**
> 
> At least for the 2 addresses above, that's not true. Please elaborate.

> 

(1) The creator of safe **0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956 (safe1)** is **0xE3e959BE4af88296ccf67134939f2609a58Eb9f0 (address1)**. The txn that transfers funds from **address1** to the **main address** is https://etherscan.io/tx/0xc3b4b922c7d0e90f32a833f84e59f89367ccdcc2cdd8c4c9fc66e895594ca3b7, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

(2) The creator of safe **0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce (safe2)** is **0x89E7781D7bdbbf92Deb5d75574188AA2fe6B54AC (address2)**. The txn that transfers funds from **address2** to the **main address** is https://etherscan.io/tx/0xb9aecf33cb1b058fec076fbed87bde23d6723e540a39b0e9c2e1fe4a64d72fbc, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

(3) The creator of safe **0xc3011a19b221d2571b94e85b3079c9092b437b73 (safe3)** is **0xE4c19E77073d9a22840DB594E6dFcF282748EDFf (address3)**. The txn that transfers funds from **address3** to the **main address** is https://etherscan.io/tx/0xd2fe66829f734f5e73d651664e28cd2cbdeeb3faaeec0a1057078e2d451cac97, which sends the funds to **0xac40b1b9affdf358ff4f268085562b1efab8a473 (main address)**.

To sum up, certain addresses used for airdrop farming might not exhibit completely same movement patterns. For example, for **safe2** and **address2**, after the txn that transfers funds from **address2** to the **main address**, the airdrop farmer used another address to transfer some ETH into **address2** and made some interactions with **safe2**. But in all, airdrop farming is obvious. I can list all safe addresses, creator addresses, and farming pattern etherscan links if you need. Let me know if you need further details.

**Comment by tschubotz** [comment]
Thanks for the report @p3rcy-eth  we've verified that the following addresses seems to be airdrop farming.

```
0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956
0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce
0xc3011a19b221d2571b94e85b3079c9092b437b73
0x11d61ca55bed7e67f509a054e68958b911e4e163
0x2a5d746c8b5b099bc32f17c0862b52dde9e4a7fa
0xf8585ee97a6edede2d6438f60b05e543dc5c703b
0xc2284bfcbceea21f1e914e67f796583d8b1cec1b
0xed922a25aa6f8ffa8fb99343e5b0c9a90c517193
0x2696b20a5e3bd09703f825d449fb973ed18315b9
0x74071154a47cb452b53d1c8940ebfe86620b515a
0x0ed9eef0d135ea684067b2cf7c9a8809200ffc4d
0xe40d6d04a6b02de42537ed437e3d4acfaed60451
0xd4b613b77c713cc9f500c05da1f4be33342e7354
0x782de266cc0524581bd6b6d0bfe3ddcab3ac1998
0x9116d17e4e2996ac620985d3c81c3df9aae9757a
0x625cc1d6280d8288ddeee4e313d309e96d3dcbdf
0x8ecab80540b4e764e65a0586d99c9ebb351ecdfe
0x54262b056000332bd82793f7154c71966010554b
0xc6d6e408dc0a01cb26ddd2f8316e9f4b3e9a1e75
0x678141b710e680380167aeb0fcbd6010213fe339
0x44d026e6952e42b2f3d6e881fb9619f6e111b63f
0x210c0642ac42962ed4609b8433f6a263d0f3d1e8
0xd3f49e496d2ba3041739fcb8a644c29ca6a1af42
0x5b332dca77ea2eb9ad281b313de071b6d14330de
0x7d4d1c22edce3d69c6212e6dad9c51e6af3137fd
0xf2444be94cc78be648f5d90b686c0e5a282cacec
0x77a8f147a78873bcd49acbc96b533440ca6c2c82
0xc33959fe31ce88fd8b4fc60579dd6ab6998fd575
0x9f08511f0d88a360801a837f47700d8207b7456d
0xcf066abe1e200919882a55aa026b7549bd78e632
0x85d150e0779b8bad89fd8885bece16f34026a7b2
0xd6e188fc0273ea597c7ba350a5d411b2c1816987
0x296428d8868d51e81fd28fcf208757713975f369
0x26cfd0af2f9d7a9cd0e05ea7d92382d9189f5897
0xcf36a04ede4f3ea8f11c524a3073d9c8dca91c5d
0x228ea0ee7c976a2b83bb7d95c3d801beefd4f866
0x29d88b0a3dfb8971e13853b6dcfcac1124e54551
0x119d4499e697a371b3804440bb6cd406dd1388db
0xb803e9bb25b5d1e0c536c18f8069e55f3f910fca
0x7cebc0abe652a2d84f5cd0aecff6604c1088a10c
0x1f21f4c79aa7b46933ca6f6811e321f4170cb296
0x7cf59d3a059d4e410091c2303d0d01b304b3c22a
0x731598be46067f3e5724571e75ec96a7e41ab8df
```

**Comment by p3rcy-eth** [comment]
> Thanks for the report @p3rcy-eth we've verified that the following addresses seems to be airdrop farming.
> 
> ```
> 0x5b5f79e6e0568a7c2b9b09ca95c5822e53ca9956
> 0x5081f72b5cec91ebbb26aeae96bb51715a4bc8ce
> 0xc3011a19b221d2571b94e85b3079c9092b437b73
> 0x11d61ca55bed7e67f509a054e68958b911e4e163
> 0x2a5d746c8b5b099bc32f17c0862b52dde9e4a7fa
> 0xf8585ee97a6edede2d6438f60b05e543dc5c703b
> 0xc2284bfcbceea21f1e914e67f796583d8b1cec1b
> 0xed922a25aa6f8ffa8fb99343e5b0c9a90c517193
> 0x2696b20a5e3bd09703f825d449fb973ed18315b9
> 0x74071154a47cb452b53d1c8940ebfe86620b515a
> 0x0ed9eef0d135ea684067b2cf7c9a8809200ffc4d
> 0xe40d6d04a6b02de42537ed437e3d4acfaed60451
> 0xd4b613b77c713cc9f500c05da1f4be33342e7354
> 0x782de266cc0524581bd6b6d0bfe3ddcab3ac1998
> 0x9116d17e4e2996ac620985d3c81c3df9aae9757a
> 0x625cc1d6280d8288ddeee4e313d309e96d3dcbdf
> 0x8ecab80540b4e764e65a0586d99c9ebb351ecdfe
> 0x54262b056000332bd82793f7154c71966010554b
> 0xc6d6e408dc0a01cb26ddd2f8316e9f4b3e9a1e75
> 0x678141b710e680380167aeb0fcbd6010213fe339
> 0x44d026e6952e42b2f3d6e881fb9619f6e111b63f
> 0x210c0642ac42962ed4609b8433f6a263d0f3d1e8
> 0xd3f49e496d2ba3041739fcb8a644c29ca6a1af42
> 0x5b332dca77ea2eb9ad281b313de071b6d14330de
> 0x7d4d1c22edce3d69c6212e6dad9c51e6af3137fd
> 0xf2444be94cc78be648f5d90b686c0e5a282cacec
> 0x77a8f147a78873bcd49acbc96b533440ca6c2c82
> 0xc33959fe31ce88fd8b4fc60579dd6ab6998fd575
> 0x9f08511f0d88a360801a837f47700d8207b7456d
> 0xcf066abe1e200919882a55aa026b7549bd78e632
> 0x85d150e0779b8bad89fd8885bece16f34026a7b2
> 0xd6e188fc0273ea597c7ba350a5d411b2c1816987
> 0x296428d8868d51e81fd28fcf208757713975f369
> 0x26cfd0af2f9d7a9cd0e05ea7d92382d9189f5897
> 0xcf36a04ede4f3ea8f11c524a3073d9c8dca91c5d
> 0x228ea0ee7c976a2b83bb7d95c3d801beefd4f866
> 0x29d88b0a3dfb8971e13853b6dcfcac1124e54551
> 0x119d4499e697a371b3804440bb6cd406dd1388db
> 0xb803e9bb25b5d1e0c536c18f8069e55f3f910fca
> 0x7cebc0abe652a2d84f5cd0aecff6604c1088a10c
> 0x1f21f4c79aa7b46933ca6f6811e321f4170cb296
> 0x7cf59d3a059d4e410091c2303d0d01b304b3c22a
> 0x731598be46067f3e5724571e75ec96a7e41ab8df
> ```

@tschubotz Hello ser, it seems that in the latest file in https://github.com/safe-global/safe-user-allocation-reports/blob/main/review/valid_reports.csv doesn't include any of the safe addresses reported in this report. Please kindly help me check it. Thanks a lot!

**Comment by tschubotz** [comment]
Fixed in https://github.com/safe-global/safe-user-allocation-reports/commit/7029772687651c66524e0ba8b5b59052f745e5ec thanks.
